apps/humanatlas.io/src/assets/content/2d-ftu-page/table-data/ftu-cell-count-7th-release.csv

```diff
@@ -0,0 +1,155 @@
+Organ,FTU Label in Uberon,FTU ID in Uberon,CT Label in CL,CT ID in CL,CT Label in 2D Object,#AS,#CT
```

Comment:
I would prefer if the headers were shorter and do not use spaces. Ex. `FTU Label in Uberon` -> `FtuLabel`, `FTU ID in Uberon` -> `FtuId`, etc.